lib/preprocessor.py

```diff
@@ -206,8 +207,14 @@ def server_start(self, ctype, sname, opts):
             if crash_expected:
                 # disable crash detector
                 self.servers[sname].crash_expected = True
-            self.servers[sname].start(silent=True, rais=True, wait=wait,
-                                      wait_load=wait_load, args=args)
+            is_started = self.servers[sname].start(
+                    silent=True, rais=True, wait=wait,
+                    wait_load=wait_load, args=args)
+            if not is_started and wait and wait_load:
+                color_stdout('\n[Instance "{0}"] Failed to start tarantool '
+                             'instance\n'.format(sname),
+                             schema='error')
+                self.kill_current_test()
```

Comment:
To be honest, I don't get, why we should do something aside catching an exception and re-raising it as `LuaPreprocessorException`. In my head, the plan was to raise TarantoolStartError, when we reach the timeout, and everything else should work.

Reply:
Fixed